chapters/VK_EXT_acquire_drm_display/acquire_drm_display.txt

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018-2021 The Khronos Group, Inc.
+//
+// SPDX-License-Identifier: CC-BY-4.0
+
+[open,refpage='vkAcquireDrmDisplayEXT',desc='Acquire access to a VkDisplayKHR using DRM',type='protos']
+--
+
+To acquire permission to directly a display in Vulkan from the Direct Rendering
+Manager (DRM) interface, call:
+
+include::{generated}/api/protos/vkAcquireDrmDisplayEXT.txt[]
+
+  * pname:physicalDevice The physical device the display is on.
+  * pname:drmFd Connection to the DRM node file descriptor.
+  * pname:display The display the caller wishes Vulkan to control.
+
+All permissions necessary to control the display are granted to the Vulkan
+instance associated with the provided pname:physicalDevice until the display is
+either released or the connector is unplugged.
+If any error is encountered during the acquisition of the display, the call
+must return the error code ename:VK_ERROR_INITIALIZATION_FAILED.
+
+The provided DRM fd should not be closed before the display is released,
+attempting to do it may result in undefined: behaviour.
```

Comment:
Typo: extra `:` here

Reply:
It's to avoid [CI failure](https://github.com/bl4ckb0ne/Vulkan-Docs/runs/2784979633?check_suite_focus=true), see https://github.com/KhronosGroup/Vulkan-Docs/blob/b4e8cd820b2487bc892b391fb26b49501473a6a6/style/writing.txt#L254

Although I am not sure if `undefined:` or `undefined{cl}` is prefered, I've seen both in other extensions.

Reply:
Oh. My bad.

Reply:
'undefined:' is a markup convention used to indicate that the term is agreed by implementers in this context, and specifies what sort of undefined (behavior | values | etc.) may occur.

'undefined{cl}' is a markup convention used only in the style guide in places where we want to present markup, not invoke the asciidoctor macro of the same name. If there are any outstanding extension proposals containing this markup, they need to be corrected. I don't see this happening in main branch.

Reply:
https://www.khronos.org/registry/vulkan/specs/1.2/styleguide.html#writing-undefined